test/elixometer_test.exs

```diff
@@ -267,4 +267,12 @@ defmodule ElixometerTest do
     assert {:error, :not_found} == get_metric_value("elixometer.test.bad.bad")
   end
 
+  test "a subscription that has additional subscription options" do
+    Application.put_env(:elixometer, :subscribe_options, [{:some_option, 42}])
+
+    update_counter("subscribe_options", 1)
+
+    assert subscription_exists "elixometer.test.counters.subscribe_options"
+    assert [{:some_option, 42}] = Reporter.options_for("elixometer.test.counters.subscribe_options")
```

Comment:
I know, shall we change it to that? I was keeping a bit to what is being done in erlang but I don't care.